.github/workflows/ci.yml

```diff
@@ -27,12 +27,20 @@ jobs:
           python-version: ${{ matrix.python-version }}
 
       - name: Test typing_extensions
+        if: matrix.python-version != "3.11-dev"
         run: |
```

Comment:
I think the Github Actions incantation might be something like:
```
continue-on-error: ${{ matrix.python-version == '3.11-dev' }}
```

The advantage being that you get to see red crosses.

Reply:
Ah you're right, I was reading something that suggested `continue-on-error` wouldn't work properly